src/sentry/migrations/0152_groupinbox_sort_index.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-01-26 01:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0151_add_world_map_dashboard_widget_type"),
+    ]
+
+    operations = [
+        migrations.AlterIndexTogether(
+            name="groupinbox",
+            index_together=set([("project", "date_added")]),
+        ),
+    ]
```

Comment:
Will ask ops to run this before I merge, ddl is
```
BEGIN;
--
-- Alter index_together for groupinbox (1 constraint(s))
--
CREATE INDEX CONCURRENTLY "sentry_groupinbox_project_id_date_added_fd440409_idx" ON "sentry_groupinbox" ("project_id", "date_added");
COMMIT;
```

Will take a while since `sentry_groupinbox` is a large table.